sources/web/datalab/polymer/components/item-list/item-list.ts

```diff
@@ -54,6 +57,16 @@ class ItemListElement extends Polymer.Element {
    */
   public columns: Array<string>;
 
+  /**
+   * Whether to hide the header row
+   */
+  public noHeader: boolean;
```

Comment:
It's easy to set the default value to true, right? Down on line 87, where the default is currently explicitly set to false.